builder/upcloud/builder.go

```diff
@@ -110,11 +117,6 @@ func (self *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (pa
 }
 
 // Cancel is called when the build is cancelled
-func (self *Builder) Cancel() {
-	if self.runner != nil {
```

Comment:
You sure this check is no longer needed? IIRC there was some reason for adding it.

Reply:
Currenlty it is handled by context in Run

Reply:
I will check for sure in the evening

Reply:
Works nice. Interrupts handled gracefully. Feel free to merge

Reply:
Cancel even removed from packer.Builder interface